x/txfees/keeper/fees.go

```diff
@@ -128,44 +131,44 @@ func ValueFromPtr[T any](ptr *T) (zero T) {
 }
 
 // swapFeeToBaseDenom swaps the taker fee coin to the base denom.
-// If the fee token is unknown, it is sent to the community pool.
+// Returns error if the fee token is unknown or if swapping fails.
 // The fee must be sent to the txfees module account beforehand.
 func (k Keeper) swapFeeToBaseDenom(
```

Comment:
`swapFeeToBaseDenom` was cleaned of the `communityPool` logic.
now handled by caller, based on returned err